pvlib/spectrum/irradiance.py

```diff
@@ -176,3 +178,93 @@ def get_reference_spectra(wavelengths=None, standard="ASTM G173-03"):
         )
 
     return standard
+
+
+def average_photon_energy(spectrum):
+    r"""
+    Calculate the average photon energy of one or more spectral irradiance
+    distributions.
+
+    Parameters
+    ----------
+    spectrum : pandas.Series or pandas.DataFrame
+
+        Spectral irradiance, must be positive. [Wm⁻²nm⁻¹]
+
+        A single spectrum must be a :py:class:`pandas.Series` with wavelength
+        [nm] as the index, while multiple spectra must be a
+        :py:class:`pandas.DataFrame` with column headers as wavelength [nm].
+
+    Returns
+    -------
+    ape : numeric or array
+        Average Photon Energy [eV].
+        Note: returns ``np.nan`` in the case of 0 Wm⁻²nm⁻¹  spectral irradiance
+        input.
+
+    Notes
+    -----
+    The average photon energy (APE) is an index used to characterise the solar
+    spectrum. It has been used widely in the Physics literature since the
+    1900s, but its application for solar spectral irradiance characterisation
+    in the context of PV performance modelling was proposed in [1]_. The APE
```

Comment:
```suggestion
    spectrum. It has been used widely in the physics literature since the
    1900s, but its application for solar spectral irradiance characterisation
    in the context of PV performance modeling was proposed in [1]_. The APE
```

Reply:
Changed Physics but left modeling
There are other British English spellings throughout the notes (my other merged PRs too) so changing this single one would be inconsistent, and... well... I'm a Brit after all, so... 😶
Unless US English is pvlib policy then let me know because I'll have a few more changes to make 😂